client/landing/gutenboarding/onboarding-block/loading-placeholder/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
```

Comment:
😱 Calypso import ⚠️ 

If we wanted to use this, I'd say we'd need to extract it to `@automattic/components`. However, I think we can use a different loading view like @simison [mentioned](https://github.com/Automattic/wp-calypso/pull/39266#issuecomment-582321330) and we shouldn't need this.

Reply:
Also [core spinner](https://github.com/WordPress/gutenberg/tree/0c089f7a481fae56dd493d679bf7b2ea000b21c4/packages/components/src/spinner) is available.

Reply:
Ohh Right... ☠️ Calypso Imports! 💀   .  😆 

Il just remove the spinner for now per @simison 's suggestion.